package.json

```diff
@@ -10,7 +10,7 @@
     "watch": "gulp watch",
     "build": "gulp transpile",
     "mocha": "mocha",
-    "e2e-test": "gulp e2e-test",
+    "e2e-test": "gulp transpile && mocha --timeout 600000 build/test/functional/",
```

Comment:
This required that `mocha` be globally installed. It would be better to be
```
"e2e-test": "gulp transpile && npm run mocha -- -t 600000 --recursive build/test/functional/",
```